src/test/ui/rfc-2632-const-trait-impl/maybe-const-trait-impl.rs

```diff
@@ -0,0 +1,9 @@
+// parse-fail
```

Comment:
Remove this. There's no such directive.

Reply:
And also this doesn't seem to be true of the test. The errors below are not parser errors.